blob/drivertest/drivertest.go

```diff
@@ -1225,6 +1225,23 @@ func testCanceledWrite(t *testing.T, newHarness HarnessMaker) {
 			if _, err := w.Write(cancelContent); err != nil {
 				t.Fatal(err)
 			}
+
+			// Verify that the previous content (if any) is still readable,
```

Comment:
It kind of feels like all the same test to me; I supposed it could be split out into "cancel" vs "read during write" but I'm not sure that's worth it.